Symfony/CS/Tests/Fixer/Symfony/MethodSeparationFixerTest.php

```diff
@@ -204,7 +323,6 @@ public function method444f1()
     }
 
     /**/
-
```

Comment:
ping @GrahamCampbell , please have a fresh look at the tests. I would like to wrap this one up and backport it to the 1.x line.